core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/AvroUpsertMonitor.scala

```diff
@@ -274,7 +274,12 @@ class AvroUpsertMonitorActor(
       }
     } yield ()
 
-    importFuture.map(_ => ImportComplete) pipeTo self
+    //Make sure message is acknowledged in the case of any failure while trying to construct importFuture
+    importFuture.map(_ => ImportComplete)  recover {
+      case _ =>
+        acknowledgeMessage(message.ackId)
```

Comment:
For the case where a workspace isn't found, it does update the import status, but it wouldn't for errors in `getPetServiceAccountUserInfo` or `getImportStatus`.  I could move the `publishMessageToUpdateImportStatus` for missing workspaces to the recover so that it covers those cases.  Same with logging.